From 6e7a0683788906bb7ae21a708d5519481d6d058f Mon Sep 17 00:00:00 2001
From: Jonathan Lebon <jonathan@jlebon.com>
Date: Wed, 20 Mar 2019 16:14:29 -0400
Subject: [PATCH] stages/files: Also relabel subuid/subgid files

Those get touched by `useradd` and so we need relabeling if we added any
users or groups.

Closes: #762
---
 internal/exec/stages/files/passwd.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/internal/exec/stages/files/passwd.go b/internal/exec/stages/files/passwd.go
index 1ae0f02..58fde2e 100644
--- a/internal/exec/stages/files/passwd.go
+++ b/internal/exec/stages/files/passwd.go
@@ -38,6 +38,8 @@ func (s *stage) createPasswd(config types.Config) error {
 			"/etc/group*",
 			"/etc/shadow*",
 			"/etc/gshadow*",
+			"/etc/subuid*",
+			"/etc/subgid*",
 			"/etc/.pwd.lock",
 			"/home",
 			"/root",
-- 
2.20.1

